src/lib.rs

```diff
@@ -1028,6 +1030,59 @@ impl Ini {
         Ini::load_from_file_opt(filename, ParseOption::default())
     }
 
+    /// Load from files;overwrite and append
+    #[cfg(feature = "case-insensitive")]
+    pub fn load_from_files<P: AsRef<Path>>(filenames: &Vec<P>) -> Result<Ini, Error> {
+        let mut merged = Ini::new();
+        let mut section_2_props: HashMap<Option<UniCase<String>>, Properties> = HashMap::new();
+        for filename in filenames {
+            match Ini::load_from_file(filename) {
+                Ok(ini) => {
+                    for (section, props) in ini.sections {
+                        if let Some(section_props) = section_2_props.get_mut(&section) {
+                            for (key, value) in props {
+                                section_props.insert(key, value);
+                            }
+                        } else {
+                            section_2_props.insert(section, props);
+                        }
+                    }
+                }
+                Err(e) => return Err(e),
+            }
+        }
+        for (section, props) in section_2_props {
+            merged.sections.insert(section, props);
+        }
+        Ok(merged)
+    }
+
+    #[cfg(not(feature = "case-insensitive"))]
+    pub fn load_from_files<P: AsRef<Path>>(filenames: &Vec<P>) -> Result<Ini, Error> {
+        let mut merged = Ini::new();
+        let mut section_2_props: HashMap<Option<String>, Properties> = HashMap::new();
```

Comment:
It doesn't need to be 2 different functions to support `case-insensitive`, just use something like:

```
cfg_if! {
    if #[cfg(feature = "case-insensitive")] {
        let mut section_2_props: HashMap<Option<String>, Properties> = HashMap::new();
    } else {
        let mut section_2_props: HashMap<Option<UniCase<String>>, Properties> = HashMap::new();
    }
}
```